infra/base-images/base-builder-jvm/Dockerfile

```diff
@@ -54,10 +55,13 @@ RUN cp $(bazel cquery --output=files //src/main/java/com/code_intelligence/jazze
     cp $(bazel cquery --output=files //deploy:jazzer-api) $JAZZER_API_PATH && \
     cp $(bazel cquery --output=files //deploy:jazzer-junit) $JAZZER_JUNIT_PATH
 
-USER root
+FROM base AS final
 
-RUN rm -rf /home/jazzer_user/.cache/bazel /home/jazzer_user/.cache/bazelisk && \
-    rm -rf $SRC/jazzer
+USER root
```

Comment:
We don't need this anymore because we changed users in  builder, not base.

Reply:
@tballison With your changes, we indeed don't need to explicitly set the user in this image anymore.